src/plugins/js-eval/run.js

```diff
@@ -22,14 +22,16 @@ const inspect = (val) => {
 };
 
 function exposeBuiltinInGlobal(name) {
-  const require2 = require;
   const setReal = (val) => {
     delete global[name];
     global[name] = val;
   };
   Object.defineProperty(global, name, {
     get: () => {
-      const value = require2(name); // eslint-disable-line
+      const value = require(name); // eslint-disable-line
+      if (name === 'timers') {
```

Comment:
for some reason, in node15 repl, timers.promises is undefined, but that's the exception, fs.promises, stream.promises, dns.promises are set